lib/svg-sprite/shape.js

```diff
@@ -149,14 +149,17 @@ function SVGShape(file, spriter) {
     this.spriter = spriter;
     this.svg = { current: this.source.contents.toString(), ready: null };
     this.name = path.basename(this.source.path);
+    // todo: get rid of setting empty object if this.spriter.config.shape is not provided
+    // https://github.com/svg-sprite/svg-sprite/pull/653#discussion_r841069781
     this.config = merge(merge({}, defaultConfig), this.spriter.config.shape || {});
-    // todo: get rid of setting empty object if this.spriter.config.shape is not provided (https://github.com/svg-sprite/svg-sprite/pull/653#discussion_r841069781)
+
+    const relativePath = this.source.base ? path.relative(this.source.base, this.source.path) : this.name;
```

Comment:
It's better be closer to where it's used (in this case, line 161)

Reply:
Yes! You're right

Reply:
Done